docs/se/guides/40_webclient.adoc

```diff
@@ -0,0 +1,416 @@
+    .path("/greet")     // <2>
+    .request(String.class)  // <3>
+    .peek(System.out::println)  // <4>
+    .await();   // <5>
+----
+<1> Create a Http GET request.
+<2> Path to reach after the base uri provided during instantiation.
+<3> Execute the request and return `Single` of the desired type.
+<4> Print the response in the console.
+<5> Wait for server response because of reactive approach.
+
+The path method joins `/greet` to the WebClient base Uri. The target Uri for this request becomes
+`http://localhost:8080/greet` where the response should be a greeting message. As the response will be printed in the
+console, a String is more convenient than the default return type for the request. The default type is WebClientResponse
+which contains response code, headers and non-handled entity.
+
+==== Run the application
+
+[source,bash]
+.Build the quickstart:
+----
+mvn package
+----
+
+This command will create helidon-quickstart-se.jar in the target folder.
+
+[source,bash]
+.Run the greeting application first:
+----
+java -cp target/helidon-quickstart-se.jar io.helidon.examples.quickstart.se.Main
+----
+
+Open a new command prompt or terminal and run the ClientExample class you just created.
+
+[source,bash]
+.Run the greeting application first:
+----
+java -cp target/helidon-quickstart-se.jar io.helidon.examples.quickstart.se.ClientExample
+----
+
+[source,bash]
+.Output:
+----
+{"message":"Hello World!"}
+----
+
+When the ClientExample finishes its execution, you can stop the Main class by pressing `ctrl+c`.
+
+==== Discover other WebClient functionality
+
+In practice, String is not the most used returned type. It could be more interesting to return an object such as
+JsonObject. Earlier, Json support was added to the webclient, so it can be used instead of String.
+
+[source,java]
+.Replace String by JsonObject:
+----
+import javax.json.JsonObject;
+
+...
+
+webClient.get()
+    .path("/greet/David")
+    .request(JsonObject.class)  // <1>
+    .peek(System.out::println)
+    .await();
+----
+<1> Request a JsonObject as return value.
+
+In the Uri, the string after `greet` is a path parameter which allows the application to greet someone.
+
+[source,bash]
+.Output:
+----
+{"message":"Hello David!"}
+----
+
+This time, a JsonObject is printed out in the console. It is possible to change the greeting itself by using a PUT
+request to `/greet/greeting` endpoint from the base Uri.
+
+[source,java]
+.Modify the application greeting:
+----
+import javax.json.Json;
+
+...
+
+JsonObject entity = Json.createObjectBuilder() // <1>
+    .add("greeting", "Bonjour")
+    .build();
+webClient.put()     // <2>
+    .path("/greet/greeting")
+    .submit(entity)     // <3>
+    .thenCompose(response -> webClient.get()    // <4>
+        .path("/greet/David")
+        .request(JsonObject.class))
+    .thenAccept(System.out::println)
+    .await();
+----
+<1> Create a JsonObject with key `greeting` and value `bonjour`.
+<2> Execute a PUT request.
```

Comment:
Create
You are executing the PUT request with the submit. Now you are just creating it.